src/logic/calculate.js

```diff
@@ -0,0 +1,120 @@
+import operate from './operate';
+
+function isNumber(item) {
+  return !!item.match(/[0-9]+/);
+}
+
+/**
+ * Given a button name and a calculator data object, return an updated
+ * calculator data object.
+ *
+ * Calculator data object contains:
+ *   total:s      the running total
+ *   next:String       the next number to be operated on with the total
+ *   operation:String  +, -, etc.
+ */
+export default function calculate(obj, buttonName) {
+  if (buttonName === 'AC') {
+    return {
+      total: null,
+      next: null,
+      operation: null,
+    };
+  }
```

Comment:
- [ ] When clicking this button `AC` please display a default `zero` in the screen even when you are not doing any operation or when you just opened an application show a default `zero` it can be a good user XP

Reply:
- This issue is well addressed :clap:.